checker/totals.go

```diff
@@ -22,6 +22,11 @@ type AggregatedScan struct {
 	MTASTSEnforceList []string
 }
 
+const (
+	TopDomainsSource = "TOP_DOMAINS"
+	LocalSource      = "LOCAL"
```

Comment:
Will this behave right if we already have scans under source lowercase `local`?

Reply:
We should have any aggregated scans under source local in prod yet. The source is set when the aggregated scan is imported, not when it occurs.